walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,96 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<TimeRange> BusyTimes = new ArrayList<TimeRange>();
```

Comment:
variables are usually given names in `camelCase`. In this case that would mean it should be `busyTimes` and `freeTimes`.  Classes are CapitalizedCamel and constants are in SCREAMING_CASE.

Reply:
Noted!